src/main/kotlin/mate/academy/service/RandomFigureSupplier.kt

```diff
@@ -0,0 +1,56 @@
+package mate.academy.service
+
+import mate.academy.model.Color
+import mate.academy.model.Figure
+import mate.academy.model.FigureType
+import kotlin.random.Random
+
+private const val MIN_SIDE: Int = 10
+private const val MAX_SIDE: Int = 100
+class RandomFigureSupplier {

```

Comment:
extra line between constants and class def
```suggestion

class RandomFigureSupplier {
```